Makefile

```diff
@@ -64,6 +64,16 @@ docker:
 	@echo ">> building docker image"
 	@docker build -t "$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)" .
 
+k8s: promu
+	@sed -e '/linux\/amd64/{n;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;d;}' $(shell pwd)/.promu.yml > $(shell pwd)/.promu4travis.yml
```

Comment:
This `sed` seems very brittle. What will happen if someone edits `.promu.yml`? What does it _do_ in the first place?